adapter/src/main/java/org/mifos/chatbot/adapter/SmallTalk.java

```diff
@@ -0,0 +1,40 @@
+package org.mifos.chatbot.adapter;
+
+import org.mifos.chatbot.core.model.Intent;
+
+class SmallTalk {
+    private static final String[] SMALL_TALK_INTENTS = {
```

Comment:
Can we move this to a static file for logical separation, and easy modification later.

Reply:
made interface SmallTalkService for that. And moved isSmallTalkRequest from SmallTalk.java to SmallTalkHandler.